Harvest.Web/ClientApp/src/Requests/RequestContainer.tsx

```diff
@@ -147,166 +147,176 @@ export const RequestContainer = () => {
   }
 
   return (
-    <div className={originalProject && "card-wrapper"}>
-      {originalProject !== undefined && (
-        <ProjectHeader
-          project={originalProject}
-          title={"Original Field Request #" + (originalProject?.id || "")}
-        />
-      )}
-      <div className={originalProject && "card-green-bg"}>
-        <div className="row justify-content-center">
-          <div className="col-md-6 card-wrapper no-green mt-4 mb-4">
-            <div className="card-content">
-              <h2>
-                {projectId ? "Create Change Request" : "Create Field Request"}
-              </h2>
-              <div className="row">
-                <div className="col-md-6">
-                  <div className="form-group">
-                    <Label>When to Start?</Label>
-                    <div className="input-group" style={{ zIndex: 9000 }}>
-                      <DatePicker
-                        format="MM/dd/yyyy"
-                        required={false}
-                        clearIcon={null}
-                        value={project.start}
-                        onChange={(date: Date) =>
-                          setProject({ ...project, start: date })
-                        }
-                      />
+    <div>
+      <div className="alert alert-info">
+        You are making a change request for {project.name}{" "}
+        <a href={`/project/details/${project.id}`} className="alert-link">
```

Comment:
@wongband We don't ever want to use `<a>` inside a react app since that doesn't cause the client-side react-router routing.